src/Text/Pandoc/Readers/HTML.hs

```diff
@@ -927,25 +927,21 @@ pMath inCase = try $ do
   let attr = toStringAttr attr'
   unless inCase $
     guard (maybe True (== mathMLNamespace) (lookup "xmlns" attr))
-  let isDisplay = case lookup "display" attr of
-                     Just "block" -> True
-                     _ -> False
+  let possiblyDisplayMath = case lookup "display" attr of
```

Comment:
Instead of `possiblyDisplayMath`, I think `constructor` would be better.

Reply:
Done.